src/python/pants/bin/pants_loader.py

```diff
@@ -96,6 +97,7 @@ def run_default_entrypoint() -> None:
 
     @classmethod
     def main(cls) -> None:
+        native_engine.initialize()
```

Comment:
This is a new method which needs to be called as early as possible to allow `grpc_util::initialize` to eventually be called so TLS algorithms can be setup in-process.